projects/storefrontapp/src/app/spartacus/spartacus-b2c-configuration.module.ts

```diff
@@ -17,6 +17,7 @@ import { environment } from '../../environments/environment';
 
 const defaultBaseSite = [
   'electronics-spa',
+  'electronics-spa-standalone',
```

Comment:
[QUESTION]
I'm lacking context and happy to learn: Why `electronics-spa-standalone` is special that we needed to add it for the OPF epic PR?

Reply:
Yes, this basesite will be dedicated for the OPF checkout testing.